More DBAPI use in manager/processors/

kp52 opened the pull request:
Updated  manager/processors/save_htmlsnippet...save_template.processor

Commit:
More DBAPI use in manager/processors/

Updated  manager/processors/save_htmlsnippet...savetemplate.processor

kp52 commented:
I will go back over the files and:
- scrap the redundant error handling
- apply **$modx->getFullTableName()** more consistently to simplify the SQL queries 
- separate the **$icon** bugfix into a separate commit

I can bear the pain of the pull request being rejected if that's the simplest way to get where we want to be

TimGS commented:
OK - the $icon issue should go to 1.1.develop. It makes life much easier if the branch DBAPI_use is only used for the conversion of mysql_* functions to $modx->db->* functions and _closely_ related issues e.g. scrapping the redundant error handling.

Branch DBAPI_use will not be merged into 1.1.develop until after the stable release of 1.1 i.e it is provisionally scheduled for v1.2.

thanks,
Tim.

Commit:
DBAPI use in manager/processors/save*

- use API calls instead of mysql_ in all the processors/save_*.php files
- remove redundant error checking on calls to $modx->db->query
- consistent use of $modx->getFullTableName to clarify SQL statements
